### ISSUE TITLE
please add a hint so that when Spring Security is on the classpath, we automatically get HTTP and HTTPS support

### ISSUE THREAD

**Issue by joshlong** [spring-projects-experimental/spring-native#1117]
@NativeHint(options = "--enable-url-protocols=http,https")

**Comment by joshlong** [comment]
Ideally, they'd be enabled by default for everything, always. I just ran Liquibase (with no HTTP server) and it needed HTTP for schema resolution.

**Comment by sdeleuze** [comment]
I don't think we should add that by default, especially https which is impacting significantly the memory footprint, but we can add flags with the right granular hints. We could decide to add https by default for `WebClient` and `RestTemplate`. For server, I am not sure since servers can be behind an http server which will take car of https.
